src/main/kotlin/minesweeper/model/Cells.kt

```diff
@@ -0,0 +1,18 @@
+package minesweeper.model
+
+import kotlin.reflect.KProperty
+
+class Cells(
+    val cells: List<Cell>
+) {
```

Comment:
아래와 같이 위임을 바꾸면 size프로퍼티를 정의할 필요없지 않을까요?

```suggestion
class Cells(
    val cells: List<Cell>
) List<Cell> by cells {
```